emcc.py

```diff
@@ -602,7 +602,10 @@ def check_human_readable_list(items):
 
 def make_js_executable(script):
   src = read_file(script)
-  cmd = shared.shlex_join(config.JS_ENGINE)
+  cmd = config.JS_ENGINE
+  if settings.WASM_BIGINT:
+    cmd.append('--experimental-wasm-bigint')
```

Comment:
I worry this flag may only exist in new-enough node... but looks like it's in `12.4.0` at least, so seems ok since the emsdk ships a newer one.

Reply:
I could revert this part of the change... since the test is now disabled on wasm64 anyway... 

Bear in mind that this code is only ever used with autoconf-based builds.